acronym/acronym.py

```diff
@@ -1,2 +1,7 @@
 def abbreviate(words):
-    
+    acronym = "".join(word[0] for word in words.split())
```

Comment:
looks good, the one thing I saw is it makes the acronym lower case, if it needs to be uppercase, we would need to change it, but for now, this looks great to me!